cypress/views/common.ts

```diff
@@ -0,0 +1,89 @@
+import { DATA_FEDERATION, DATA_SERVICES } from '../constants/common';
+
+export const MCGMSCommon = {
+  visitMCGMSDashboard: () => {
```

Comment:
this is a standard mcg-mc-console operation we can keep in the separate mcg-ms console file. 

the remaining are related general console which is other than our operator.

what you say, should we keep a separate these?

Reply:
ok